src/operations/update.ts

```diff
@@ -55,7 +85,11 @@ class UpdateOneOperation extends CommandOperation {
   }
 }
 
-class UpdateManyOperation extends CommandOperation {
+export interface UpdateManyOperationOptions extends CommandOperationOptions {
+  multi?: boolean;
```

Comment:
this should not be an option, `UpdateMany` => `multi: true`, `UpdateOne` => `multi: false`